recipes/ogre/1.x/conanfile.py

```diff
@@ -0,0 +1,284 @@
+        "OGRE_BUILD_COMPONENT_JAVA": False,
+        "OGRE_BUILD_COMPONENT_MESHLODGENERATOR": True,
+        "OGRE_BUILD_COMPONENT_OVERLAY": True,
+        "OGRE_BUILD_COMPONENT_OVERLAY_IMGUI": True,
+        "OGRE_BUILD_COMPONENT_PAGING": True,
+        "OGRE_BUILD_COMPONENT_PROPERTY": True,
+        "OGRE_BUILD_COMPONENT_PYTHON": True,
+        "OGRE_BUILD_COMPONENT_RTSHADERSYSTEM": True,
+        "OGRE_BUILD_COMPONENT_TERRAIN": True,
+        "OGRE_BUILD_COMPONENT_VOLUME": True,
+        "OGRE_BUILD_DEPENDENCIES": False,
+        "OGRE_BUILD_PLUGIN_BSP": True,
+        "OGRE_BUILD_PLUGIN_DOT_SCENE": True,
+        "OGRE_BUILD_PLUGIN_OCTREE": True,
+        "OGRE_BUILD_PLUGIN_PCZ": True,
+        "OGRE_BUILD_PLUGIN_PFX": True,
+        "OGRE_BUILD_PLUGIN_STBI": True,
+        "OGRE_BUILD_RENDERSYSTEM_D3D11": True,
+        "OGRE_BUILD_RENDERSYSTEM_GL": True,
+        "OGRE_BUILD_RENDERSYSTEM_GL3PLUS": True,
+        "OGRE_BUILD_RTSHADERSYSTEM_SHADERS": True,
+        "OGRE_BUILD_SAMPLES": True,
+        "OGRE_BUILD_TESTS": False,
+        "OGRE_BUILD_TOOLS": True,
+        "OGRE_CONFIG_ENABLE_QUAD_BUFFER_STEREO": False,
+        "OGRE_CONFIG_FILESYSTEM_UNICODE": True,
+        "OGRE_CONFIG_THREADS": 3,
+        "OGRE_CONFIG_THREAD_PROVIDER": "std",
+        "OGRE_ENABLE_PRECOMPILED_HEADERS": True,
+        "OGRE_INSTALL_PDB": True,
+        "OGRE_INSTALL_SAMPLES": True,
+        "OGRE_INSTALL_TOOLS": True,
+        "OGRE_INSTALL_VSPROPS": False,
+        "OGRE_NODELESS_POSITIONING": True,
+        "OGRE_PROFILING_REMOTERY_PATH": "",
+        "OGRE_RESOURCEMANAGER_STRICT": 2,
+        "OGRE_STATIC": True,
+        "OGRE_SET_DOUBLE": False,
+        "OpenEXR_Half_LIBRARY": "OpenEXR_Half_LIBRARY-NOTFOUND",
+        "OpenEXR_Half_LIBRARY_DEBUG": "OpenEXR_Half_LIBRARY_DEBUG-NOTFOUND",
+        "OpenEXR_INCLUDE_DIR": "OpenEXR_INCLUDE_DIR-NOTFOUND",
+        "OpenEXR_Iex_LIBRARY": "OpenEXR_Iex_LIBRARY-NOTFOUND",
+        "OpenEXR_Iex_LIBRARY_DEBUG": "OpenEXR_Iex_LIBRARY_DEBUG-NOTFOUND",
+        "OpenEXR_IlmImf_LIBRARY": "OpenEXR_IlmImf_LIBRARY-NOTFOUND",
+        "OpenEXR_IlmImf_LIBRARY_DEBUG": "OpenEXR_IlmImf_LIBRARY_DEBUG-NOTFOUND",
+        "OpenEXR_IlmThread_LIBRARY": "OpenEXR_IlmThread_LIBRARY-NOTFOUND",
+        "OpenEXR_IlmThread_LIBRARY_DEBUG": "OpenEXR_IlmThread_LIBRARY_DEBUG-NOTFOUND",
+        "Qt5_DIR": "Qt5_DIR-NOTFOUND",
+        "SDL2_DIR": "${ogre-build-dir}/Dependencies/cmake",
+        "pugixml_DIR": "${ogre-build-dir}/Dependencies/lib/cmake/pugixml"
+    }
+    exports_sources = "CMakeLists.txt", "patches/**"
+    _cmake = None
+
+    def requirements(self):
+        self.requires("freeimage/3.18.0")
+        self.requires("boost/1.75.0")
+        self.requires("freetype/2.11.1")
+        self.requires("openexr/2.5.7")
+        self.requires("poco/1.11.2")
+        self.requires("tbb/2020.3")
+        self.requires("zlib/1.2.12")
+        self.requires("zziplib/0.13.71")
+        self.requires("openssl/1.1.1o", override=True)
```

Comment:
```suggestion
        self.requires("openssl/1.1.1o", override=True)
        self.requires("xorg/system")
```
Maybe, it should solve your error with X11 dependency

Reply:
Thanks. Fingers crossed :crossed_fingers: 
added manually since we need to add the depenency in `self.cpp_info.components["OgreMain"].requires`